src/Microsoft.AspNet.Mvc.Xml/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="EnumerableWrapperProvider_InvalidSourceEnumerableOfT" xml:space="preserve">
     <value>The type must be an interface and must be or derive from '{0}'.</value>
   </data>
+  <data name="RequiredProperty_MustHaveDataMemberRequired" xml:space="preserve">
+    <value>Value type property '{0}' on type '{1}' has RequiredAttribute but no DataMember(IsRequired = true) attribute.</value>
```

Comment:
Needs to explain why this is important and why a user should care.

"blah blah blah is a problem when used for ____. Do ______ to fix this" 

Please fill in the blanks.